src/test/skript/tests/syntaxes/expressions/ExprMetadata.sk

```diff
@@ -0,0 +1,21 @@
+test "metadata value":
```

Comment:
could we please get some tests for adding/removing/setting to incorrect typed and null values